.github/actions/install-rust/main.js

```diff
@@ -0,0 +1,38 @@
+const child_process = require('child_process');
```

Comment:
do you really need node for this? it can be done with just a shell script

Reply:
To be honest, I don't know anything about these test files. I just got them from the secret labs contract template.